.github/workflows/pr.yml

```diff
@@ -143,6 +143,13 @@ jobs:
           sonar_utility: sonar-scanner
           sonar_commands: '("-Dsonar.projectKey=${{env.SONAR_PROJECT_KEY}} -Dsonar.host.url=${{env.SONAR_HOST_URL}} -Dsonar.login=${{env.SONAR_LOGIN}} -Dsonar.pullrequest.key=${{github.event.number}} -Dsonar.pullrequest.branch=${{github.event.pull_request.head.ref}} -Dsonar.pullrequest.base=${{github.event.pull_request.base.ref}}")'
 
+      - name: FrogBot
+        uses: lumada-common-services/gh-composite-actions@stable
+        env:
+          JF_URL: https://hitachi.jfrog.io/
```

Comment:
This should be `https://one.hitachivantara.com`

Reply:
"I haven't tested FrogBot with this URL. I will test and update.

Reply:
updated the URL